Helper/FeatureSwitch/Definitions.php

```diff
@@ -665,5 +670,11 @@ class Definitions
             self::DEFAULT_VAL_KEY => false,
             self::ROLLOUT_KEY     => 0
         ],
+        self::M2_DISABLE_BOLT_FOR_CUSTOMER_GROUP => [
+            self::NAME_KEY        => self::M2_DISABLE_BOLT_FOR_CUSTOMER_GROUP,
```

Comment:
Let's rename to something like "M2_ALLOW_DISABLING_BOLT_FOR_CUSTOMER_GROUP'.
With the current name, readers may be under the impression that something will be immediately disabled, but actually the feature switch only enable this option